frontend/src/static/js/v_authorship.js

```diff
@@ -165,12 +184,21 @@ window.vAuthorship = {
           }), {});
     },
 
+    filesSort() {
```

Comment:
naming wise, should follow existing style. 
action follow by nouns.
E.g. sortFiles,